src/style.css

```diff
@@ -0,0 +1,110 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: content-box;
+}
+
+#container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  margin: 5% 10%;
+  border: 1px solid #e7e6e6;
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2);
+}
+
+button:hover {
+  cursor: pointer;
+}
+
+.heading {
+  display: flex;
+  justify-content: space-between;
```

Comment:
You are using display flex many times. I think it would be better if you declare a class name, add display: flex; property and use it whenever you want in order to avoid repeating your self-concept.

Reply:
Thanks for the suggestions